core/src/main/java/com/sopt/core/designsystem/component/timetable/NoostakTimeTable.kt

```diff
@@ -0,0 +1,188 @@
+package com.sopt.core.designsystem.component.timetable
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.sopt.core.designsystem.theme.NoostakTheme
+import com.sopt.domain.entity.TimeTableEntity
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.time.format.TextStyle
+import java.util.Locale
+
+@Composable
+fun NoostakTimeTable(
+    data: TimeTableEntity,
+    modifier: Modifier = Modifier
+) {
+    val days = data.timeEntity.size
+    val timeSlots = calculateTimeSlots(data.startTime, data.endTime)
+
+    LazyVerticalGrid(
+        modifier = modifier
+            .border(
+                width = 1.dp,
+                color = NoostakTheme.colors.gray200,
+                shape = RoundedCornerShape(8.dp)
+            ),
+        columns = GridCells.Fixed(days + 1)
+    ) {
+        items((days + 1) * (timeSlots + 1)) { index ->
+            val rowIndex = index / (days + 1)
+            val columnIndex = index % (days + 1)
+
+            val cellType = determineCellType(rowIndex, columnIndex)
+            val backgroundColor = getBackgroundColor(cellType, rowIndex, columnIndex, data)
+            val text = getCellText(cellType, rowIndex, columnIndex, data)
+
+            NoostakTimeTableBox(
+                index = index,
+                days = days,
+                timeSlots = timeSlots,
+                backgroundColor = backgroundColor,
+                text = text
+            )
+        }
+    }
+}
+
+@Composable
+fun getBackgroundColor(
+    cellType: CellType,
+    rowIndex: Int,
+    columnIndex: Int,
+    data: TimeTableEntity
+): Color {
+    return when (cellType) {
+        CellType.Blank, CellType.DateHeader, CellType.TimeHeader -> Color.Transparent
+        CellType.Data -> {
```

Comment:
P2:  return 생략하고 요렇게 표현식으로도 쓸 수 있을 것 같아요~!

```suggestion
): Color = when (cellType) {
    CellType.Blank, CellType.DateHeader, CellType.TimeHeader -> Color.Transparent
    CellType.Data -> {
```

Reply:
내 코틀린 센세가 되.